### ISSUE TITLE
Request: Increase the nonce for unconfirmed transactions to allow for signing multiple transactions in the same block

### ISSUE THREAD

**Issue by carumusan** [polkadot-js/apps#3698]
This is more relevant for blockchains with long confirmation times such as Kulupu (60s). Not increasing the nonce causes transactions other than the first to error.

![image](https://user-images.githubusercontent.com/879525/93364910-0a719500-f841-11ea-9003-1bdd371410a7.png)

**Comment by jacogr** [comment]
So the transaction layer on the API actually support both "nonce-from-chain" (the default above) and "nonce-with-pending-txs". The UI has not exposed this as of now.

It is also something I bump into quite often, have been thinking about the default going toward the nextNonce (with txpool taken into account), possibly with an option for change on the signer dialog.

TL;DR A very valid enhancement request, UI just needs to expose the functionality.
